### DISCUSSION TITLE
Surface Conditions in Coupled Runs

### DISCUSSION THREAD

**Discussion by megandevlan** [NCAR/amwg_dev#150] [TF5: Coupled SE/MOM6]
In light of large differences between coupled cases, primarily 009 and 016 (all changes we want to include vs. the new control case) @cecilehannay suggested we take a look at the near-surface conditions simulated in CAM as well. As a first pass, here are a few time series of both global annual averages and over the Labrador Sea in particular:
<img width="1088" alt="Screen Shot 2022-08-01 at 10 51 30 AM" src="https://user-images.githubusercontent.com/42625059/182201869-d13658af-b40b-49da-bb1e-686ac941c4ed.png">
 
<img width="1006" alt="Screen Shot 2022-08-01 at 10 51 56 AM" src="https://user-images.githubusercontent.com/42625059/182201928-8b56d897-1cb7-48d1-a321-f33d22166ff7.png">

Runs 18b and 19b are similar to 017 but have a more reasonable RESTOM (after some tuning of CLUBB parameters). For right now they're somewhat short, but it looks like this is already having an impact on global mean LHFLX. In the Labrador Sea in particular though, it's interesting to note that after 25-30 years, the SHFLX and LHFLX in case 009 is much lower than in the new control and subsequent tests; so changing the dynamical core doesn't reproduce that behavior by this metric. It'll probably be worth taking a look at seasonal means (not just annual), and maybe looking at some of these differences spatially as well though.

**Comment by swrneale** [comment]
@megandevlan  this is great. And I think it gives hope that 017 is not freezing as LHFLX and SHFLX remains elevated at year 30 much more like 016 than any of the other runs. Could you also plot for the cloud forcings (SWCF/LWCF)? Thanks.

**Reply by megandevlan** [reply]
Agreed, I think it's safe to say (for now) that run 017 isn't freezing the same way, and so far 018b and 019b look to be pretty well behaved too: 
![Screen Shot 2022-08-02 at 11 06 37 AM](https://user-images.githubusercontent.com/42625059/182433266-ebd81c4e-fc99-42a0-ba1d-118f7dfaec65.png)

Both the SWCF and LWCF are pretty different over the Labrador Sea in 009 and 016, but given the different surface fluxes this probably makes sense. Related, it doesn't look like the latest runs have a steady decline in 2m temperature like we see in 009, which seems like a good sign too. 

Looking spatially seems to highlight the point of concern pretty well too, but again I don't think the difference between FV and SE cases (016 being FV and others being SE) is driving the big difference from 009 -  
![Screen Shot 2022-08-02 at 11 45 33 AM](https://user-images.githubusercontent.com/42625059/182440050-b6d4fe59-7f8b-4986-8336-e0c10eb0a2c8.png)
![Screen Shot 2022-08-02 at 11 45 54 AM](https://user-images.githubusercontent.com/42625059/182440111-04761072-ec45-4585-9266-69bc3e5819d7.png)

**Reply by swrneale** [reply]
What time periods are these for as it would matter whether ice has formed or not yet?

**Reply by JulioTBacmeister** [reply]
Thanks Meg - This is a great analysis.   It really does seem that 016-019b are on a different trajectory than 009.  It's a bit puzzling to me that 019b Lab Sea convection in Gustavo's looks so weak (#149).   I agree with Rich that 017 looks in good shape.

**Reply by megandevlan** [reply]
Good point Rich, that will definitely matter. These maps are just a mean over all months for as many years are available per case. I'm hoping to look at this seasonally too - maybe a time mean over just DJF? Taking time slices could also be useful, so looking at years 10-20 vs. 20-30 might help as well. 

@JulioTBacmeister, it's definitely interesting that 019b Lab Sea convection is so weak. I'm not sure I see why based on any of these variables. But I'm also looking mostly at annual means, I wonder if I increase the resolution to monthly (seems like this is what the convection plot is showing) if something will pop out as more obvious. I'll give that a go too and see if something becomes more clear.

**Reply by megandevlan** [reply]
It's also possible my Lab Sea definition isn't totally exact; I'll see if I can grab a better mask, mine's only roughly defined in this region (plotted SHFLX just to get a feel for the area):
![Screen Shot 2022-08-02 at 12 52 49 PM](https://user-images.githubusercontent.com/42625059/182451557-5f81c249-e1bb-4b6a-a368-412a4c2d1591.png)

**Comment by megandevlan** [comment]
Following on the discussion in (https://github.com/NCAR/amwg_dev/issues/149), I'm taking a similar approach to @gustavo-marques and plotting the mean March fields for years 15-25 in all cases. The surface fluxes in 0017 are weaker in that region of large MLD compared to 018b and 019b:
![Screen Shot 2022-08-02 at 3 30 01 PM](https://user-images.githubusercontent.com/42625059/182476816-2748156b-ded9-4743-b004-a1f450b5b4f3.png)

It looks like the near-surface winds (neutral equivalent U10 in this case) are weaker in the 017 case around the southern edge of Greenland, maybe that's factoring in (understandably, looks like largest MLDs are correlated with where U10n is large)
![Screen Shot 2022-08-02 at 3 32 07 PM](https://user-images.githubusercontent.com/42625059/182477095-2703c096-e6b3-4313-a6d7-2f7a9c171be5.png)

In terms of cloud forcing, 018b and 019b bumped back up the LWCF relative to 017 thanks to the tuning, but it's stronger further into the Lab Sea in 018b if I'm seeing this right:
![Screen Shot 2022-08-02 at 3 40 33 PM](https://user-images.githubusercontent.com/42625059/182478320-5d79e303-8ed6-48ac-93cc-aa1ff4863367.png)

**Reply by swrneale** [reply]
Any chance you could just add ICEFRAC alongside these?

**Reply by megandevlan** [reply]
Definitely! Years 15-25, March mean - it does look like 017 and 019b are forming a bit more ice than the 018b case. The latter looks like it could be poised to stay close to 016. 
<img width="1042" alt="Screen Shot 2022-08-02 at 6 53 33 PM" src="https://user-images.githubusercontent.com/42625059/182501214-9f3d8f62-2502-4a3d-805a-2ba2f3db5865.png">

**Reply by swrneale** [reply]
It's very interesting that even though in yrs 15-25 run 017 has minimal deep convection similar to 009, it still has quite a lot of open ocean similar to  016.

**Comment by megandevlan** [comment]
I updated these plots for years 25-35 in each case rather than 15-25; it looks like run 017 still has plenty of open ocean, so it's not freezing over and neither are the 18b and 19b runs. But the surface fluxes are pretty different in 017 with a lot less LHFLX in the Lab Sea. Maybe this is contributing to the salinity bias there, making the water fresher if less is evaporating? I'd imagine that could play a role in the convection. 
![Screen Shot 2022-08-04 at 9 13 20 AM](https://user-images.githubusercontent.com/42625059/182883092-5738cc51-e0f8-4342-bb4b-380264beb98f.png)
![Screen Shot 2022-08-04 at 9 13 33 AM](https://user-images.githubusercontent.com/42625059/182883144-c8c594bd-13cb-4831-a16a-45df0977cd1c.png)
![Screen Shot 2022-08-04 at 9 14 01 AM](https://user-images.githubusercontent.com/42625059/182883251-f8d4b5f4-4277-409d-a588-f76504ab78fc.png)
![Screen Shot 2022-08-04 at 9 15 00 AM](https://user-images.githubusercontent.com/42625059/182883445-fa81a3e7-e763-4fec-bbec-2539cae71f11.png)

**Reply by swrneale** [reply]
How about another one :) ? Could you plot TS masked for the ocean only, just to see whether SHFLX is obviouslyimpacting the near-surface ocean temperature in each run.

**Reply by megandevlan** [reply]
This is more of a limited plot for TS (everything in white either land or outside the colorbar range), but It seems like the weaker SHFLX in 017 does also have slightly cooler near-surface temperatures too, at least compared to 018b and 019b: 
![Screen Shot 2022-08-04 at 2 26 37 PM](https://user-images.githubusercontent.com/42625059/182946127-b5a09fb9-11ec-4db9-a3fd-550dc355776a.png)

At least that sounds like it's in line with what we know about the runs - 017 stays warmer than 009 certainly, but it's still possibly a bit cooler than the others. This is all kind of tightly linked, but the near-surface winds are a bit weaker in U17 as well (based on the neutral equivalent 10m that's output by default, at least):  
![Screen Shot 2022-08-04 at 2 33 13 PM](https://user-images.githubusercontent.com/42625059/182947260-f066e499-635c-4142-b8df-896fe9bfed13.png)

**Comment by megandevlan** [comment]
Quick look at the global mean surface values with 9 years of run 020 included (020 = 018b but using camdev phys). These are just the annual mean values, but for some reason global mean SHFLX looks like it's unusually low and LWCF is pretty weak as well: 

<img width="1040" alt="Screen Shot 2022-08-05 at 9 00 18 AM" src="https://user-images.githubusercontent.com/42625059/183104681-be012956-b5a9-4128-b27d-0a0650631fc3.png">
